app/views/records/by_filter_task.html.erb

```diff
@@ -0,0 +1,62 @@
+<h1>Expected Result Patient List</h1>
+<div class="pull-right button-row">
+  <% if @measure && @measure.sub_id %>
+    <div class="btn-group">
+      <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">Sub measures <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu">
+        <% submeasures = HealthDataStandards::CQM::Measure.where(hqmf_id: @measure.hqmf_id).sort_by(&:sub_id) %>
+        <% submeasures.each do |submeasure| %>
+          <li><%= link_to "#{submeasure.display_name} - #{submeasure.subtitle}", product_test_task_by_filter_task_path(@task.product_test, @task, sub_id: submeasure.sub_id),  method: :get %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <%= button_to good_results_task_path(@task), :method => :get, :class => "btn btn-default" do %>
+    <i class="fa fa-fw fa-download" aria-hidden="true"></i> Download HTML Patients
+  <% end %>
+</div>
+
+<% cache [@records, @measure, hide_patient_calculation?] do %>
+  <div class="row">
+    <div id="records_list" class="col-sm-12">
+      <%= render partial: 'records_list', locals: { records: @records } %>
+    </div>
+  </div>
+<% end #cache records %>
+
+<script type="text/javascript">
```

Comment:
This seems to be a copy paste from `app/views/records/index.html.erb`, would it be possible to move this out into a proper JS file so this same javascript code can be reused?

Reply:
Actually it looks like there is no `#search_measures` block on this page at all, would it be possible to just remove the whole javascript block?